src/_posts/databases/postgresql/2000-01-01-accessing.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Accessing Your Scalingo for PostgreSQL® Addon
+nav: Accessing
+modified_at: 2023-12-01 00:00:00
+tags: databases postgresql addon
+index: 6
+---
+
+
+Remotely accessing your Scalingo for PostgreSQL® database can sometimes be
```

Comment:
question: I find the expression "Scalingo for PostgreSQL®" hard to read in the text. Is it mandatory or could we just write "PostgreSQL®"?

Reply:
It seems Aiven use this "typology" at the beginning of the document and then uses a simplified version: https://docs.aiven.io/docs/products/postgresql

Reply:
I wanted to add some marketing in there :smiling_face_with_tear: 
Alright, I'll remove it in a next commit.

Reply:
Should be fixed by https://github.com/Scalingo/documentation/pull/2389/commits/7d7a6697e4aa7450141c28a56ccd2e74e718dd15